deps.edn

```diff
@@ -1,18 +1,25 @@
 {:paths   ["src" "classes"]
 
  :deps
-          {org.clojure/clojure               {:mvn/version "1.10.1"}
-           clj-time                          {:mvn/version "0.14.2"}
-           javax.servlet/servlet-api         {:mvn/version "2.5"}
-           org.apache.activemq/activemq-core {:mvn/version "5.7.0"}
-           org.slf4j/slf4j-simple            {:mvn/version "1.7.30"}
-           amperity/vault-clj                {:mvn/version "0.5.1"}
-           org.apache.commons/commons-email  {:mvn/version "1.5"}
-           com.taoensso/timbre               {:mvn/version "4.10.0"}}
+          {org.clojure/clojure                   {:mvn/version "1.10.1"}
+           org.clojure/data.json                 {:mvn/version "1.0.0"}
+           clj-time                              {:mvn/version "0.14.2"}
+           cljfmt                                {:mvn/version "0.6.7"}
+           javax.servlet/servlet-api             {:mvn/version "2.5"}
+           org.apache.activemq/activemq-core     {:mvn/version "5.7.0"}
+           org.apache.tika/tika-core             {:mvn/version "1.24"}
+           org.slf4j/slf4j-simple                {:mvn/version "1.7.30"}
+           amperity/vault-clj                    {:mvn/version "0.5.1"}
+           org.apache.commons/commons-email      {:mvn/version "1.5"}
+           com.taoensso/timbre                   {:mvn/version "4.10.0"}
+           com.google.cloud/google-cloud-storage {:mvn/version "1.108.0"}}
 
- :aliases {:test     {:extra-paths ["test"]
-                      :extra-deps  {lambdaisland/kaocha {:mvn/version "1.0.629"}}
-                      :main-opts   ["-m" "kaocha.runner"]}
-           :uberdeps {:extra-deps {uberdeps {:mvn/version "0.1.10"}}
-                      :main-opts  ["-m" "uberdeps.uberjar"]}}
+ :aliases {:test      {:extra-paths ["test"]
+                       :extra-deps  {lambdaisland/kaocha {:mvn/version "1.0.629"}
+                                     com.google.cloud/google-cloud-nio {:mvn/version "0.5.0"}}
+                       :main-opts   ["-m" "kaocha.runner"]}
+           :uberdeps  {:extra-deps {uberdeps {:mvn/version "0.1.10"}}
+                       :main-opts  ["-m" "uberdeps.uberjar"]}
+           :lint      {:main-opts ["-m" "cljfmt.main" "check"]}
+           :format    {:main-opts ["-m" "cljfmt.main" "fix"]}}
```

Comment:
I'm not familiar with the differences between cljfmt and eastwood but it'll probably be easier to use the same one for both PTC & WFL!

Reply:
Thanks, that makes sense! My understanding is that `eastwood` is just a linter (think about flake8), while `cljfmt` can both lint and format (think about yapf or black), which is why I went down with `cljfmt`. What do you think?